src/bin/tree-sitter-graph/main.rs

```diff
@@ -138,7 +138,7 @@ fn main() -> Result<()> {
     let mut functions = Functions::stdlib();
     let mut config = ExecutionConfig::new(&mut functions, &globals_).lazy(lazy);
     let graph = file
-        .execute(&tree, &source, &mut config)
+        .execute(&tree, source_path, &source, tsg_path, &tsg, &mut config)
```

Comment:
Showing excerpts requires the path to the file, the contents of the file, and a location. We get the location dynamically, and the other two are passed as static parts of the execution context.

Finally, we show excerpts from both the .tsg file and the target file, so we need to pass both in.